### PR TITLE
block controls UI cleanup

### PR DESCRIPTION
**Pull request by kaitlinbolling**
Cleans up the block controls and styles the manual posts selector panel.
Note: [hiding whitespace](https://github.com/alleyinteractive/wp-curate/pull/29/files?diff=unified&w=1) will be helpful when reviewing.

### CODE REVIEW

File: blocks/query/edit.tsx [hunk 1]
```diff
@@ -276,82 +277,75 @@ export default function Edit({
           initialOpen
         >
           {minNumberOfPosts !== undefined && minNumberOfPosts !== maxNumberOfPosts ? (
-            // @ts-ignore
-            <PanelRow>
+            <>
               { /* @ts-ignore */ }
               <RangeControl
                 label={__('Number of Posts', 'wp-curate')}
+                help={__('The maximun number of posts to show.', 'wp-curate')}
```
Comment (jessicamgoddard) [inline blocks/query/edit.tsx]: small type (maximum) :)

File: blocks/query/edit.tsx [hunk 2]
```diff
@@ -276,82 +277,75 @@ export default function Edit({
...
-              value={postTypes}
-              onChange={(newValue) => setAttributes({ postTypes: newValue })}
-              options={displayTypes}
-              multiple
-            />
-          </PanelRow>
-          { /* @ts-ignore */ }
+          <RangeControl
+            label={__('Offset', 'wp-curate')}
+            help={__('The number of post to pass over.', 'wp-curate')}
```
Comment (jessicamgoddard) [inline blocks/query/edit.tsx]: "post" should be "posts"

File: blocks/query/index.scss [hunk 3]
```diff
@@ -2,6 +2,44 @@
...
  */
 
-.wp-curate-post-type-selector .components-base-control {
-  width: 100%;
+.autocomplete__component .components-base-control__label {
+  font-size: 11px;
+  font-weight: 500;
+  line-height: 1.4;
+  text-transform: uppercase;
+  display: inline-block;
```
Comment (jessicamgoddard) [inline blocks/query/index.scss]: should these be alphabetical?

**Review by jessicamgoddard — APPROVED** [review]
a few small notes :)

✨

File: blocks/query/edit.tsx (hunk 2 above)
Comment (kaitlinbolling) [reply blocks/query/edit.tsx]: 😅 thanks for catching these!

File: blocks/query/index.scss (hunk 3 above)
Comment (kaitlinbolling) [reply blocks/query/index.scss]: hmm yes! Stylelint didn't yell at me. This is extending our shared stylelint config, where I guess we don't define property order.

**Pull request merged**